starboard/extension/max_input_size.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_EXTENSION_MAX_INPUT_SIZE_H_
+#define STARBOARD_EXTENSION_MAX_INPUT_SIZE_H_
+
+#include "starboard/drm.h"
+#include "starboard/media.h"
+#include "starboard/player.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define kStarboardExtensionMaxInputSizeName \
+  "dev.starboard.extension.MaxInputSize"
```

Comment:
I think we need a better extension name, maybe "enhanced_player_creation"?

Reply:
Done.